README.md

```diff
@@ -13,6 +13,10 @@ Since restarting your access points removes any scripts connected via `hostapd_c
 
 Simple `opkg install hass` once it is added to the OpenWRT repositories. Until then, download a package from releases and `opkg install <downloaded_file>`. Then you can modify `/etc/config/hass` to your liking and start/enable the service via `service hass start` and `service hass enable`.
 
+### Authentication
+
+due to auth changes in home assistant there are 2 ways of authing with home assistant, the prefered option is a long lived token that can be generated. There are 2 options, and it will prefer `token` if both are configured
```

Comment:
My proposal:

Due to auth changes in Home Assistant there are 2 authentication mechanisms, the prefered option is a [long lived token](https://developers.home-assistant.io/docs/en/auth_api.html#long-lived-access-token) that can be generated from the UI. If both are configured `token` auth will be prefered, since API password auth is deprecated.